src/cript/nodes/util/__init__.py

```diff
@@ -292,9 +293,54 @@ def _is_node_field_valid(node_type_list: list) -> bool:
         return False
 
 
-def load_nodes_from_json(nodes_json: str):
+def load_nodes_from_json(nodes_json: str) -> UUIDBaseNode:
```

Comment:
This type hint is only kind of true.

Suppose your JSON is a list of nodes, like
```
[
 { node: ["Material"], ...},
 {node: ["Collection"], ...},
 ...
 ]
 ```
 
 Then you would get a list with the types `List[cript.Material, cript.Collection, ...]`.
 And this will work for they two container types JSON supports, list and dict. 

So the function is a bit more flexible then the type hint suggests.

Reply:
I don't think we use that functionality anywhere yet though.

Reply:
hmm I see what you're saying, but not sure how to resolve it other than putting all nodes in a union for the return type. I can also happily just leave it out and try to explain in the docstrings. What are your thoughts?

Reply:
yeah, as long as we don't enforce it with @beartype I am ok with that.

Reply:
or, just remove it and explain the the docs.

Reply:
I made changes to the documentation, let me know if it is satisfactory or not. Feel free to also update the docs as well if you can quickly word it better.